app/views/jaf-repository/details.njk

```diff
@@ -0,0 +1,168 @@
+{% extends '_layout.njk' %}
+
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Home",
+        href: "/"
+      },
+      {
+        text: "JAF Repository",
+        href: "/jaf-repository"
+      },
+      {
+        text: "Comparison",
+        href: `/jaf-repository/compare/`
+      },
+      {
+        text: "Details",
+        href: "#"
+      }]
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">JAF Comparison</h1>  
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <dl class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            Base JAF name
+          </dt>
+          <dd class="govuk-summary-list__value">
+            {{ jafComparison.name }}
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            Compared JAF name
+          </dt>
+          <dd class="govuk-summary-list__value">
+            {{ jafComparison.comparedJafs.name }}
+          </dd>
+        </div>
+      </dl>
+      <table class="govuk-table" id="task-table">
+        <caption class="govuk-table__caption govuk-table__caption--m">Tasks</caption>
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Task</th>
+            <th scope="col" class="govuk-table__header govuk-table__header--numeric" aria-sort="none">
+              <button>Importance</button>
+            </th>
+            <th scope="col" class="govuk-table__header govuk-table__header--numeric" aria-sort="none">
+              <button>Frequency</button>   
+            </th>
+          </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+          {%for task in jafComparison.comparedJafs.comparison_response.tasks %}
```

Comment:
Need to have a better way of accessing these properties. In your model, think about what you need to access and design it around that. :)